src/Icicle/Sea/FromAvalanche/Program.hs

```diff
@@ -32,7 +32,7 @@ import           Icicle.Sea.FromAvalanche.Prim
 import           Icicle.Sea.FromAvalanche.State
 import           Icicle.Sea.FromAvalanche.Type
 
-import           P
+import           P hiding (head)
```

Comment:
I wanted to use it as a variable name (for buffer)

Fwiw, I don't even know why we have head in P??